security/middleware.py

```diff
@@ -1021,22 +1044,25 @@ class P3PPolicyMiddleware(BaseMiddleware):
 
     def __init__(self, get_response=None):
         super().__init__(get_response)
-        warnings.warn((
-            'DEPRECATED: The middleware "{name}" will no longer be '
-            'supported in future releases of this library.'
-        ).format(name=self.__class__.__name__))
+        warnings.warn(
```

Comment:
It has been deprecated for a while so we should drop this too.

Reply:
Updated - https://github.com/sdelements/django-security/pull/97/commits/a3af3b829cc03630cc8cf49cdca32c827c1a2f7c